recipes/adlfs/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: adlfs
+  version: {{ version }}
+
+source:
+  fn: adlfs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/a/adlfs/adlfs-{{ version }}.tar.gz
+  sha256: 71862ccad2d9e37f68b24efd9fb72d8c930667c8abf99fa73df0d3373c00d129
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - azure-datalake-store >=0.0.46,<0.1
+    - azure-storage-blob >=12.0.0
+    - fsspec >=0.6.0
+    - requests >=2.22.0,<3.0
+    - versioneer >=0.18,<1.0
```

Should it be?
```suggestion
    - python >=3
    - azure-core >=1.5.0
    - azure-datalake-store >=0.0.46,<0.1
    - azure-storage-blob >=12.0.0
    - fsspec >=0.6.0
```
And ``azure-core`` is not on ``conda-forge`` you will need to add this recipe first

If you want to, next time you can try to use [Grayskull :skull: ](https://github.com/marcelotrevisani/grayskull) to generate recipes for packages on ``pypi``  :)

---

https://github.com/conda-forge/azure-core-feedstock